kubernetes/kubernetes.go

```diff
@@ -289,6 +290,17 @@ func (in *K8SClient) GetPodLogs(namespace, name string, opts *core_v1.PodLogOpti
 	return &PodLogs{Logs: buf.String()}, nil
 }
 
+func (in *K8SClient) GetPodProxy(namespace, name, path string) ([]byte, error) {
+	return in.k8s.CoreV1().RESTClient().Get().
+		Timeout(10 * time.Second).
```

Comment:
Minor, I remember there were other timeouts for authentication endpoints IIRC perhaps worth to put the value in a constant and exposed to configuration, but not a blocker, just a side comment to future iterations.

Reply:
Good! I've used the `DefaultTimeout` value already defined for the http utils.